docs/docs/how_to/how-to-oracles.md

```diff
@@ -265,8 +265,6 @@ const app = express();
 app.use(cors())
 ```
 
-:::
```

Comment:
did you mean to delete this? wont this put the rest of the page in the `tip` block

Reply:
Ooops deleted too much here, and too little in the versioned docs lol